app/models/miq_widget/report_content.rb

```diff
@@ -28,10 +28,14 @@ def generate(user_or_group)
       end
     end
 
+    empty_row = "<tr><td>#{_("No records found")}</td>"
+    [*1..headers.length-1].each { empty_row += "<td></td>" }
+    empty_row += "</tr>"
```

Comment:
Prefer `<<` over `+=` in Ruby for string building.  However, we can also use the `*` operator here to trim it up even more:

```suggestion
    empty_row = "<tr><td>#{_("No records found")}</td>#{"<td></td>" * (headers.length - 1)}</tr>"
```